mitodl/mitxonline
# 751-flexible-pricing-remove-thank-you-page

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/mitxonline/issues/751

#### What's this PR do?
After a user submits a Flexible Price request (http://<BASE_URL>/courses/course-v1:edX+E2E-101+course/flex-price/), the user should be presented with either the `application_processing_text` or `application_approved_text` (the latter if the request was auto-approved).  Currently, regardless of whether the user is auto-approved or not, they are presented with a thank you message.

#### How should this be manually tested?

1. Submit a Flexible Price request for a user which is known to require manual approval.  Confirm that you receive the `application_processing_text`.
2. Submit a different Flexible Price request for a user which is known to be auto-approved.  Confirm that you receive the `application_approved_text`.

#### Where should the reviewer start?
https://github.com/mitodl/hq/discussions/133
